core/tests/ts-integration/tests/fees.test.ts

```diff
@@ -295,9 +440,17 @@ async function killServerAndWaitForShutdown(provider: zksync.Provider) {
 
 async function setInternalL1GasPrice(
```

Comment:
This is no longer only about internal l1 gas price, let's rename to `setFeeParams`